src/main/java/lotto/domain/LottoTicketGroup.java

```diff
@@ -25,33 +26,30 @@ private static int getTicketCount(int money) {
 
     private static void validatePays(int money) {
         if (money < LOTTO_TICKET_PRICE) {
-            throw new IllegalArgumentException("1000원 이하로는 로또를 구매할 수 없습니다.");
+            throw new PurchaseLottoTicketException("1000원 이하로는 로또를 구매할 수 없습니다.");
         }
 
-        if (money % 1000 != 0) {
-            throw new IllegalArgumentException("로또는 1000원 단위로만 구매할 수 있습니다.");
+        if (money % LOTTO_TICKET_PRICE != 0) {
+            throw new PurchaseLottoTicketException("로또는 1000원 단위로만 구매할 수 있습니다.");
         }
     }
 
     private static List<LottoTicket> generateLottoTicketGroup(int ticketCount, NumberGenerator numberGenerator) {
         List<LottoTicket> lottoTickets = new ArrayList<>();
-
         for (int i = 0; i < ticketCount; i++) {
             lottoTickets.add(new LottoTicket(numberGenerator.generateLottoNumbers()));
         }
-
         return lottoTickets;
     }
 
     public List<LottoTicket> getTickets() {
         return this.lottoTicketGroup;
     }
 
-    public RankGroup getLottoRankings(LottoTicket winningLotto) {
+    public RankGroup getLottoRankings(WinningLotto winningLotto) {
```

Comment:
```suggestion
    public Ranks getLottoRankings(WinningLotto winningLotto) {
```

취향차이 이기는한데, Group 대신에 s 를 붙이는 형태로 일급 컬렉션을 표현하면 어떨까 제안해봅니다.
LottoTicketGroup -> LottoTickets
RankGroup -> Ranks
정말로 취향 차이이므로 꼭 반영하지 않아도 괜찮습니다.